pkg/config/config.go

```diff
@@ -104,9 +104,11 @@ type HarvesterConfig struct {
 	ServerURL string `json:"serverUrl,omitempty"`
 	Token     string `json:"token,omitempty"`
 
-	OS             `json:"os,omitempty"`
-	Install        `json:"install,omitempty"`
-	RuntimeVersion string `json:"runtimeVersion,omitempty"`
+	OS                     `json:"os,omitempty"`
+	Install                `json:"install,omitempty"`
+	RuntimeVersion         string `json:"runtimeVersion,omitempty"`
+	HarvesterChartVersion  string `json:"harvesterChartVersion,omitempty"`
+	MonitoringChartVersion string `json:"monitoringChartVersion,omitempty"`
```

Comment:
Why do we want these options in the harvester config? They seem to be fixed for each Harvester release.

Reply:
The purpose was to test different RKE2 versions from PXE boot installation.
If they are not required, we can drop them.

Reply:
For `HarvesterChartVersion`, a release of Harvester will have a version string different from `0.0.0-dev` ([REF: v0.3.0](https://github.com/harvester/harvester/blob/842a9942011fa8d113897b910efd1fec9271666e/deploy/charts/harvester/Chart.yaml#L3)), so we need to inject this value into `ManagedChart` for it to work correctly.

As for `MonitoringChartVersion`, this was a variable in `build-bundle` script ([REF](https://github.com/harvester/harvester-installer/blob/be59d22082c0a0de4789ffba8cf6113ebc7d1233/scripts/build-bundle#L20)) so it can be changed during build time. To keep this behavior, I add it here so to inject the version into the `ManagedChart`.